raft/raft.go

```diff
@@ -513,10 +513,10 @@ func stepLeader(r *raft, m pb.Message) {
 		r.bcastAppend()
 	case pb.MsgAppResp:
 		if m.Reject {
-			raftLogger.Infof("%x received msgApp rejection(lastindex: %d) from %x for index %d",
+			raftLogger.Debugf("%x received msgApp rejection(lastindex: %d) from %x for index %d",
 				r.id, m.RejectHint, m.From, m.Index)
 			if pr.maybeDecrTo(m.Index, m.RejectHint) {
-				raftLogger.Infof("%x decreased progress of %x to [%s]", r.id, m.From, pr)
+				raftLogger.Debugf("%x decreased progress of %x to [%s]", r.id, m.From, pr)
```

Comment:
can we leave this one `Infof`? This one will not be repeated and tells us the change on progress.